### PR TITLE
[[FEAT]] Add support for ES2019 opt. catch param

### PR DESCRIPTION
**Pull request by jugglinmike**

### CODE REVIEW

File: src/jshint.js [hunk 1]
```diff
@@ -707,13 +707,15 @@ var JSHINT = (function() {
...
           case "9":
+          case "10":
             state.option.moz = false;
             state.option.esversion = +val;
             break;
           case "2015":
           case "2016":
           case "2017":
           case "2018":
+          case "2019":
```
Comment (caitp) [inline src/jshint.js]: side note: I wonder if we should do something similar to babel, marking individual features as allowed/disallowed, and having modules containing presets based on spec versions. These numbers mean less each year in the new living standard model.

**Comment by rwaldron** [comment]
@jugglinmike I only quickly looked at the failures and noted that they are definitely coming from the tests themselves

**Comment by jugglinmike** [comment]
@rwaldron Right you are. I've pushed up a correction

File: src/jshint.js (hunk 1 above)
Comment (jugglinmike) [reply src/jshint.js]: @caitp When you say "modules," do you mean "npm modules" (i.e. published separately) or "Node.js modules" (i.e. published within this project)?

**Comment by rwaldron** [comment]
- Rebased fixup!
- Merged into v2.11.0: https://github.com/jshint/jshint/commit/ec71bc06e8c5b0db168958423e0cf30aa01726c4

**Pull request closed without merging**